demo/index.html

```diff
@@ -37,13 +34,13 @@ <h2>
            Full window, contain, click the numbers to open:
         </h2>
         <div id="example0">
-            <a class="chocolat-image" href="demo-images/5.jpg" title="HO">
+            <a class="chocolat-image" href="demo/demo-images/5.jpg" title="HO">
```

Comment:
Why did you change these paths ? 
demo/index.html is meant to be openned directly in the browser without running a http server, like this : 
`file://path/to/demo/index.html`

Reply:
They weren't working for me when I started a new webserver at the root folder of the project. I reverted this